x/participation/keeper/msg_participate.go

```diff
@@ -32,7 +32,7 @@ func (k msgServer) Participate(goCtx context.Context, msg *types.MsgParticipate)
 	}
 
 	// check if auction is already started
-	if auction.IsAuctionStarted(blockTime) {
+	if auction.ShouldAuctionStarted(blockTime) {
```

Comment:
I believe from the API name change that we are required now to check also the status, am I right?

Reply:
This could require a bit more work for the two messages (and tests) in participation.

Reply:
I think the logic remains the same

Reply:
The logic is the same, the function name was just changed

Reply:
I disagree. Although the logic remains the same, the API name-change is clear. Having `true` returned from this function indicates only that the auction *should* be started, not that it **has** started. Anywhere where we need to be really sure that it is the case, we should check the status (and not just this API).

I personally would consider this relevant, but I guess if both of you think differently, I'm fine with that.

Reply:
The logic inside the function is the same though.  They just changed the name.  This is because to actually check the status of the auction, you need to get the `Status` field.  

So perhaps we should open an issue to check if the `Status` field is correct, instead of this one.  These functions (`ShouldAuctionStarted`, etc) are used in the `fundraising` EndBlocker to modify the actual `Status` field of auctions.

Reply:
Yeah, I think we should